core/scrollbar.js

```diff
@@ -599,7 +599,8 @@ Blockly.Scrollbar.prototype.setVisible = function(visible) {
  * @private
  */
 Blockly.Scrollbar.prototype.onMouseDownBar_ = function(e) {
-  this.onMouseUpHandle_();
+  Blockly.Touch.clearTouchIdentifier(); // This is really a click.
```

Comment:
The behaviour we want here is basically click behaviour--that when they tap on the background of the scrollbar (which is why it's onMouseDownBar_ instead of onMouseDownHandle_) we move the handle over to scroll on screen width, and don't expect it to be a drag.

There are no valid drag actions to start from here.